test/main_test.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:dart_human_crontab/dart_human_crontab.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('strict parsing', () {
+    test('throws on empty string', () {
+      try {
+        HumanCrontab.parse('');
+        assert(false);
```

Comment:
Tests should use `expect` and not `assert`
[pub.dev/packages/test](https://pub.dev/packages/test)

Reply:
Updated 👍